src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -21,6 +25,7 @@ export interface MessageData {
   content: React.ReactNode;
   index: number;
   readStatusText?: string;
+  attributes: any;
```

Comment:
```suggestion
  attributes: { [attr: string]: string};
```

I _think_ this would work, and would be a slightly better type.